src/sentry/static/sentry/app/components/installPromptBanner.jsx

```diff
@@ -41,11 +40,11 @@ const InstallPromptBanner = createReactClass({
   },
 
   sentFirstEvent() {
-    let {onboardingTasks} = this.props.organization;
-    let firstEventTask = onboardingTasks.find(task => task.task === 2);
-    this.setState({
-      sentFirstEvent: firstEventTask && firstEventTask.status === 'complete',
-    });
+    let {projects} = this.props.organization;
+    let hasFirstEvent = projects.find(
```

Comment:
You could shorten this to 

```suggestion
    return projects.find(
```

or this to hint that you're expecting a boolean value to be returned

```suggestion
    return !!projects.find(
```